executor/index_merge_reader_test.go

```diff
@@ -170,3 +170,75 @@ func (s *testSuite1) TestPartitionTableRandomIndexMerge(c *C) {
 		tk.MustQuery("select /*+ USE_INDEX_MERGE(tpk, a, b) */ * from tpk where " + cond).Sort().Check(result)
 	}
 }
+
+func (s *testSuite1) TestIndexMergeInTransaction(c *C) {
```

Comment:
We could add some `for update` read and `set transaction isolation level read committed;` test cases, the result shoue be the same if there's no concurrency.

Reply:
done